src/components/presentational/TimeSeriesChart/TimeSeriesChart.stories.tsx

```diff
@@ -68,17 +68,16 @@ function getRandomInt(min: number, max: number) {
     return Math.floor(Math.random() * (max - min + 1)) + min;
 }
 
-function getRandomIntradayWithinDomainMultipointData(start: Date) {
+function getRandomIntradayWithinDomainMultipointData(start: Date, domainMin: number = 0, domainMax: number = 200) {
     var responses = [];
     let currentTime = new Date(start);
     currentTime.setHours(0, 0, 0, 0);
     let endTime = add(currentTime, { hours: 24 });
     while (currentTime < endTime) {
         responses.push({
             timestamp: currentTime,
-            key1: (getRandomInt(0, 200)),
-            key2: (getRandomInt(0, 200)),
-            key3: (getRandomInt(0, 200)),
+            key1: (getRandomInt(domainMin - 6, domainMax + 15)),
```

Comment:
Are these value adjustments just to add a little variation in the data limits for the two keys?

Reply:
Yeah - this is to demonstrate that the two lines *don't* have the exact same min and max, and yet the gradient is applied correctly to both.  Without this, with the amount of data generation across one day, the two lines end up basically always being the same.

Reply:
Also, probably not a big deal, but you may want to adjust the default values of the `domainMin` and `domainMax` parameters, since this will result in an effective range of `-6` to `215` in the default scenario.

Reply:
This was actually on purpose, because I wanted to demonstrate that passing a smaller `domainMin` and `domainMax` than the range of the graph actually constrainted the y-axis to what you'd asked (that `allowDataOverflow` flag).

This sort of relates to @reneefromhold's questions above... and predicting what a user would expect from this base widget.  To me, if you pass a domainMin/domainMax it means you're *trying* to enforce that range because you aren't asking for auto.  However, I can see an argument of "I want the Y-Axis to go to *at least* 200, but go higher if the data is out of range".  Right now I'm constraining it and not allowing for that case (which would require passing `allowDataOverflow` to be configurable.

Reply:
Ah, yeah, makes sense.  👍